detekt/detekt.yml

```diff
@@ -1,162 +1,3 @@
-        active: false
-    DestructuringDeclarationWithTooManyEntries: # too aggressive
-        active: true
-        maxDestructuringEntries: 5
-
-formatting:
-    MaximumLineLength: # ktlint - handled by detekt
-        active: false
-    MultiLineIfElse:
-        active: false
-    ArgumentListWrapping: # ktlint - way too aggressive
-        active: false
-    NoBlankLineBeforeRbrace: # pedantic
-        active: false
-    NoConsecutiveBlankLines: # pedantic
-        active: false
-    NoEmptyFirstLineInMethodBlock: # pedantic
-        active: false
-    ParameterListWrapping: # pedantic, can be useful in compact code
-        active: false
-    CommentSpacing: # handled by custom rule
-        active: false
-    SpacingBetweenDeclarationsWithAnnotations: # nah
-        active: false
-    SpacingBetweenDeclarationsWithComments: # also nah
-        active: false
-    ImportOrdering: # handled by custom rule
-        active: false
-    AnnotationSpacing: # handled by custom rule
-        active: false
-    FunctionReturnTypeSpacing:
-        maxLineLength: 140
-    ParameterWrapping:
-        maxLineLength: 140
-    PropertyWrapping:
-        maxLineLength: 140
-    Wrapping:
-        maxLineLength: 140
-
-complexity:
-    CyclomaticComplexMethod: # default threshold of 15, caught almost every complex method
-        active: true
-        threshold: 25
-    LongParameterList: # too aggressive, classes can need a lot of params
-        active: false
-    NestedBlockDepth: # too aggressive
-        active: false
-    TooManyFunctions: # ktlint - also way too aggressive by default (11 on all file types)
-        active: true
-        thresholdInFiles: 15
-        thresholdInClasses: 20
-        thresholdInInterfaces: 20
-        thresholdInObjects: 20
-        thresholdInEnums: 11
-        ignoreAnnotated:
-            - 'SkyHanniModule'
-    ComplexCondition: # aggressive by default, at a complexity of 4
-        active: true
-        threshold: 6
-    LongMethod: # default max length of 60, caught way too much
-        active: true
-        threshold: 100
-
-exceptions:
-    SwallowedException: # there are valid reasons to do this
-        active: false
-    ThrowingExceptionsWithoutMessageOrCause: # again, valid reasons
-        active: false
-    TooGenericExceptionCaught: # sometimes you just need to catch Exception
-        active: false
-    TooGenericExceptionThrown: # we don't have our own custom exceptions
-        active: false
-
-naming:
-    ConstructorParameterNaming: # pedantic
-        active: false
-
-potential-bugs:
-    DoubleMutabilityForCollection: # went crazy about all the mutable collections
-        active: false
-    HasPlatformType: # false positives on config get() methods
-        active: false
-    StorageNeedsExpose:
-        active: true
-    NonStorageDoesntNeedExpose:
-        active: true
-    ImmutableTypesWithExpectedInteriorMutabilityInConfig:
-        active: true
-
+detekt {
```

Comment:
????

Reply:
if the file is valid yml or not, the pr works by fixing the detetk errors. since this is only bandage and no permament fix, and more a moral arguemtn and a testing stuff i can switch branches off to when gralde cant work anymore, i think its fine to leave it as is